[2024-09-29T12:36:07.720+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:36:07.734+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-19T00:00:00+00:00 [queued]>
[2024-09-29T12:36:07.739+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-19T00:00:00+00:00 [queued]>
[2024-09-29T12:36:07.741+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:36:07.761+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2024-09-19 00:00:00+00:00
[2024-09-29T12:36:07.769+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=19781) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:36:07.770+0000] {standard_task_runner.py:72} INFO - Started process 19787 to run task
[2024-09-29T12:36:07.772+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v01', 'postgres_to_s3', 'scheduled__2024-09-19T00:00:00+00:00', '--job-id', '855', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpno6otq1t']
[2024-09-29T12:36:07.775+0000] {standard_task_runner.py:105} INFO - Job 855: Subtask postgres_to_s3
[2024-09-29T12:36:07.791+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:36:07.865+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-19T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:36:07.867+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:36:08.011+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v01' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-19T00:00:00+00:00'
[2024-09-29T12:36:08.038+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:36:08.096+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:36:08.133+0000] {dag_with_postgres_hooks.py:29} INFO - Saved orders dat in text file get_rders.txt
[2024-09-29T12:36:08.139+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:36:08.169+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:36:08.171+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v01, task_id=postgres_to_s3, run_id=scheduled__2024-09-19T00:00:00+00:00, execution_date=20240919T000000, start_date=20240929T123607, end_date=20240929T123608
[2024-09-29T12:36:08.242+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:36:08.275+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T12:36:08.278+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-09-29T12:41:17.058+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:41:17.079+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-19T00:00:00+00:00 [queued]>
[2024-09-29T12:41:17.086+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-19T00:00:00+00:00 [queued]>
[2024-09-29T12:41:17.088+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:41:17.101+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2024-09-19 00:00:00+00:00
[2024-09-29T12:41:17.106+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=21178) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:41:17.107+0000] {standard_task_runner.py:72} INFO - Started process 21184 to run task
[2024-09-29T12:41:17.112+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v01', 'postgres_to_s3', 'scheduled__2024-09-19T00:00:00+00:00', '--job-id', '884', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpq_9zsb_o']
[2024-09-29T12:41:17.114+0000] {standard_task_runner.py:105} INFO - Job 884: Subtask postgres_to_s3
[2024-09-29T12:41:17.132+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:41:17.160+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2024-09-19T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:41:17.163+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:41:17.238+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v01' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-19T00:00:00+00:00'
[2024-09-29T12:41:17.239+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:41:17.253+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:41:17.266+0000] {dag_with_postgres_hooks.py:29} INFO - Saved orders dat in text file get_rders.txt
[2024-09-29T12:41:17.268+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:41:17.277+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:41:17.278+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v01, task_id=postgres_to_s3, run_id=scheduled__2024-09-19T00:00:00+00:00, execution_date=20240919T000000, start_date=20240929T124117, end_date=20240929T124117
[2024-09-29T12:41:17.329+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:41:17.339+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T12:41:17.340+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
